api/Cache.json

```diff
@@ -93,7 +93,7 @@
               "notes": "Requires HTTPS from version 33."
             },
             "safari": {
-              "version_added": "11"
+              "version_added": false
```

Comment:
It looks to me that the inconsistency in this file is flipped: the root `Cache` feature is `false` for Safari when it should be truthy (like its descendants), at least based on local testing in Safari 12